app/components/error-wrapper.js

```diff
@@ -27,22 +54,49 @@ export default Ember.Component.extend({
     }
   }),
 
+  /**
+    Determines if an error is a 404 status or not.
+
+    @property is404
+    @type Boolean
+  */
   is404: Ember.computed('httpStatusCodes', function() {
     return this.get('httpStatusCodes').contains(404);
   }),
 
+  /**
+    Updates the background based on the error class.
+
+    @property observeErrorClass
+    @type error-wrapper.updateBackground
```

Comment:
@rileytaylor I also think there is some uncertainty around what the `@type` of this property should be. *I think* it should be `@type Ember.observer` as it is an observer on a property. Maybe others can chime in here: @JoshSmith @thomascchen ?